.github/actions/github-release/README.md

```diff
@@ -0,0 +1,18 @@
+# github-release
+
+An action used to publish GitHub releases for `wasmtime`.
```

Comment:
Please mention that the original version comes from wasmtime instead. Or maybe extract it into a separate repo to be shared by wasmtime and ra?